manufacturing-ai-suite/industrial-edge-insights-time-series/helm/templates/nginx.yaml

```diff
@@ -63,10 +66,18 @@ spec:
         - name: nginx-cert-gen
           mountPath: /docker-entrypoint-initdb.d/nginx-cert-gen.sh
           subPath: nginx-cert-gen.sh
+        - name: vol-temp-nginx
+          mountPath: /opt/intel/certs
```

Comment:
As it's a third party container, let's avoid usage of intel in the path, can we stick to /opt/nginx/certs instead.
Same applies to helm templates.